demo/test.cc

```diff
@@ -39,3 +45,34 @@ TEST(Demo, OK) {
   EXPECT_EQ(200, static_cast<int>(response.code));
   EXPECT_EQ("OK\n", response.body);
 }
+
+TEST(Demo, NoPointsYet) {
+  Singleton<DemoServer>();
+  const auto response = HTTP(GET("localhost:2015/demo_id"));
+  EXPECT_EQ(200, static_cast<int>(response.code));
+  EXPECT_EQ("{\"state\":{\"points\":[]}}\n", response.body);
+}
+
+TEST(Demo, AddThreePoints) {
+  Singleton<DemoServer>();
+  // TODO(dkorolev): Support JSON parsing on the server side, like this:
+  // HTTP(POST("localhost:2015/demo_id", JSON(State::Point(+0.25, -0.25, true)), "application/json")).body);
+  // TODO(dkorolev): Should be able to HTTP(POST(...)) JSON-s directly, w/o cerealize::JSON(), right?
+  // TODO(dkorolev): How about POST-s without HTML body? Do we need those extra parameters?
```

Comment:
If you do `POST`s, you should pass the data (the point values, the label) in the request body, not in the query string. And there you could have:
* "raw JSON" with appropriate `Content-Type: application/json` header set on the request (e.g. `{x:-0.25,y:0.25,label:"1"}`) 
* or a "JSON inside a url-encoded parameter" (e.g. `data=%7B%22x%22%3A-0.25%2C%22y%22%3A0.25%2C%22label%22%3A%221%22%7D`) 
* or just a set of url-encoded values (e.g. `x=0.25&y=-0.25&label=1`) if we want to handle a simple HTML form without JavaScript around it.